src/App.vue

```diff
@@ -324,23 +389,115 @@ export default {
       }
     },
 
+    // FUTURE: update this when API returns New Incorporation task (#3102)
+    /** Gets NR data from Namex API. */
+    async getNrData (): Promise<any> {
+      await this.intializeNameXToken()
+      return this.queryNameRequest(this.nrNumber)
+    },
+
+    storeNrData (data: any): void {
+      if (!this.isNrValid(data)) {
+        this.nameRequestInvalidDialog = true
+        throw new Error('Invalid NR data')
+      }
+      if (!this.isNrConsumable(data)) {
+        this.nameRequestInvalidDialog = true
+        throw new Error('NR not consumable')
+      }
+
+      // FOR DEBUGGING ONLY
```

Comment:
FYI, this is because the expiration data in our test NRs is several months ago :]